src/app/pages/system/advanced/advanced-settings.component.html

```diff
@@ -5,18 +5,21 @@
   gdColumns="50% 50%"
   gdColumns.lt-lg="50% 50%"
   gdColumns.lt-md="auto"
-  gdAreas="data-card1 data-card2 | sysctl-card data-card3 | cron-card initshutdown-card"
-  gdAreas.lt-lg="data-card1 data-card2 | sysctl-card data-card3 | cron-card initshutdown-card"
-  gdAreas.lt-md="data-card1 | data-card2 | sysctl-card | data-card3 | cron-card | initshutdown-card"
+  gdAreas="data-card1 data-card2 | sysctl-card data-card3 | cron-card initshutdown-card | data-card7 ."
```

Comment:
dot at the end is a typo?

Reply:
It's for CSS grid. To specify empty cell in the CSS grid with a dot. The row wasn't showing up otherwise.

Reply:
Thanks for the clarification 👍🏻